DoctrWho.Db/Repositories/AuthorRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DoctorWho.Db.Interface;
+using EFCore;
+using EfDoctorWho;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DoctorWho.Db.Repositories
+{
+    public class AuthorRepository : lAuthor
+    {
+        private readonly  DoctorWhoContext _context;
+        public AuthorRepository(DoctorWhoContext context)
+        {
+            _context = context;
+        }
+
+        public  void CreateAuthor(string AuthorName)
```

Comment:
for future work, (for the final project) make the methods used to fetch DB async and using await when working with DB